src/fides/api/alembic/migrations/versions/f9b28f36b53e_privacy_preference_v2.py

```diff
@@ -214,12 +233,44 @@ def upgrade():
             "tcf_preferences", postgresql.JSONB(astext_type=sa.Text()), nullable=True
         ),
     )
+    logger.info("Added new columns to privacypreferencehistory.")
+
+    op.drop_index(
+        op.f("ix_privacypreferencehistory_preference"),
+        table_name="privacypreferencehistory",
+    )
+    op.create_index(
+        op.f("ix_privacypreferencehistory_preference_old_column"),
+        "privacypreferencehistory",
+        ["preference"],
+        unique=False,
+    )
```

Comment:
so is this simply to change the name of the index and free up that namespace for the index to be created on the new column (i.e. in L263-L268)? just wanna make sure i'm following!

Reply:
OK so to recap - this operation is now needed because we now have 2 columns (old + new) in place of one, and we want to make sure the new column index is named as we expect - so we need to delete and recreate the old index, since we still need that index, but need to allow the index on the _new_ column to be named appropriately. right?

my first instinct was to just create the new index on the new column with a different name, to save us these drop + re-create index operations here on the old column, but then we'd still need to adjust that new index name on the new column at _some_ point so that it's consistent with our mainline codebase ... so may as well just do that now.

i think that all makes sense, just talking myself through it out loud to verify! :)

Reply:
yes this what I was thinking @adamsachs, the goal to get this index name updated as part of this migration while we were taking downtime